tez-runtime-internals/src/main/java/org/apache/tez/runtime/task/TaskRunner2Callable.java

```diff
@@ -28,6 +30,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.apache.hadoop.fs.statistics.IOStatisticsContext.getCurrentIOStatisticsContext;
```

Comment:
nit: if we import some classes from org.apache.hadoop.fs.statistics, couldn't we import them in the same way? is there a specific reason for static importing this method?

Reply:
I changed it, somehow my IDE did that automatically...